presto-sqlserver/src/main/java/io/prestosql/plugin/sqlserver/SqlServerClient.java

```diff
@@ -387,4 +426,28 @@ public void setNull(PreparedStatement statement, int index)
             }
         };
     }
+
+    private static class DataCompressionDao
+    {
+        private final Handle handle;
+
+        private DataCompressionDao(Handle handle)
+        {
+            this.handle = requireNonNull(handle, "handle is null");
+        }
+
+        Optional<DataCompression> getDataCompression(JdbcTableHandle table)
+        {
+            return handle.createQuery("" +
+                    "SELECT data_compression_desc FROM sys.partitions p INNER JOIN sys.tables t ON p.object_id = t.object_id " +
```

Comment:
Could you please make this query more vertical than horizontal or share your thoughts why is it so onliner?
Maybe `FROM` and `INNER JOIN` from new line?